book/src/plumbing/jars_and_ingredients.md

```diff
@@ -205,13 +198,13 @@ The implementation of this method is defined by the `#[salsa::db]` macro; it sim
 {{#include ../../../components/salsa-2022-macros/src/db.rs:create_jars}}
 ```
 
-This implementation for `create_jar` is geneated by the `#[salsa::jar]` macro, and simply walks over the representative type for each salsa item and ask *it* to create its ingredients
+This implementation for `create_jar` is geneated by the `#[salsa::jar]` macro, and simply walks over the representative type for each salsa item and asks *it* to create its ingredients
 
 ```rust,ignore
 {{#include ../../../components/salsa-2022-macros/src/jar.rs:create_jar}}
 ```
 
 The code to create the ingredients for any particular item is generated by their associated macros (e.g., `#[salsa::tracked]`, `#[salsa::input]`), but it always follows a particular structure.
-To create an ingredient, we first invoke `Routes::push` which creates the routes to that ingredient and assigns it an `IngredientIndex`.
-We can then invoke (e.g.) `FunctionIngredient::new` to create the structure.
+To create an ingredient, we first invoke `Routes::push`, which creates the routes to that ingredient and assigns it an `IngredientIndex`.
+We can then invoke (e.g.) [Missing an example here] `FunctionIngredient::new` to create the structure.
```

Comment:
Looks like we're missing an example here.

Reply:
Yes, that is what it is referring to. The idea is that it's the `new` method of whichever ingredient is being created, I guess.